### DISCUSSION TITLE
Which settings will result in the best performance ?

### DISCUSSION THREAD

**Discussion by risedangel** [xoureldeen/Vectras-VM-Android#18] [Q&A]
Hello, thank you for this amazing app. I am new to emulation world and i would like to now more.
I think that enabling more ram and 4 coures of cpu in the settings will result in better performance but how important other settings are ?, what is the importance of Cpu Model and should i enable kvm and mttcg and etc, any help would be appreciated .

**Comment by xoureldeen** [comment]
use only mttcg and 4 cpu cores
cpu model just changing cpu cpu name and can fix some graphical issues

for windows use core2duo and linux use qemu cpu
